department-of-veterans-affairs/vets-api
# [VI-371] Add mhv account creation service

## Summary
- Add new MHV account creation service. 
- Uses Sign-in service STS auth
- Add `sign_in_service` gem for

## Related issue(s)
- https://jira.devops.va.gov/browse/VI-371

## Testing 
- STS token call can be tested locally
   - seed database: `rails db:seed`
   - start server - `rails s`
   - run in rails console - `rails c`
       ```ruby
       require 'mhv/account_creation/service'

       MHV::AccountCreation::Service.new.send(:config).sts_token(user_identifier: 'some-identifier')
       ```

      - You should get a token
- MHV api testing needs to take place on staging.

## What areas of the site does it impact?
MHV account creation
